--- Libraries/xcformatter/Sources/DefaultFormatter.cpp	2021-02-21 09:04:22.583229511 +0200
+++ Libraries/xcformatter/Sources/DefaultFormatter.cpp.new	2021-02-21 09:04:17.487000000 +0200
@@ -237,7 +237,7 @@
 
         if (invocation.showEnvironmentInLog()) {
             std::map<std::string, std::string> sortedEnvironment = std::map<std::string, std::string>(invocation.environment().begin(), invocation.environment().end());
-            for (std::pair<std::string, std::string> const &entry : sortedEnvironment) {
+            for (auto const &entry : sortedEnvironment) {
                 message += INDENT + "export " + entry.first + "=" + entry.second + "\n";
             }
         }
